src/graphics/program-lib/chunks/combineDiffuseSpecularNoRefl.frag

```diff
@@ -1,4 +1,3 @@
 vec3 combineColor() {
     return dAlbedo * dDiffuseLight + dSpecularLight * dSpecularity;
 }
```

Comment:
Just revert this file. You actually want the newline at the end because it formats constructed shaders better anyway.

Reply:
reverted